tools/test/audio/process_test.m

```diff
@@ -139,15 +139,18 @@
 	fprintf('Number of skipped tests = %d\n', r.n_skipped);
 	fprintf('============================================================\n');
 
-	if r.n_fail > 0 || r.n_pass < 1
+	n_fail = r.n_fail;
+	n_pass = r.n_pass;
+	n_na = r.n_na;
+	if r.n_fail < 1 && r.n_pass < 1
```

Comment:
I've noticed that in Matlab comparing floats to integers is not accurate, so better to use `<` instead of `==`. I think the same could happen in C when mixing floats and integers. It could be avoided by using fixed types in Matlab for fixed nature variables but it also adds the variables declarations and less compact code.

Reply:
Oh, yeah, double is default in matlab